airflow/config_templates/default_celery.py

```diff
@@ -39,16 +39,16 @@ def _broker_supports_visibility_timeout(url):
 DEFAULT_CELERY_CONFIG = {
     'accept_content': ['json'],
     'event_serializer': 'json',
-    'worker_prefetch_multiplier': conf.getint('celery', 'worker_prefetch_multiplier', fallback=1),
+    'worker_prefetch_multiplier': conf.getint('celery', 'worker_prefetch_multiplier'),
```

Comment:
We still need a fallback because existing configs out there may not have this set.

Reply:
Airflow falls back to the defaults shipped with it, so even if they don't have it, when this code lands it'll still end up being 1.

Reply:
(e.g. https://github.com/apache/airflow/blob/4ad21f5f7c2d416cf813a860564bc2bf3e161d46/airflow/configuration.py#L446)

Reply:
Ah, you’re right. (There are one too many layers of changes for me to follow here 😓)